docs/Reference.md

```diff
@@ -18,12 +18,67 @@ Discord exposes different versions of our API. You can specify which version to
 ###### API Versions
 
 | Version | Status       | Default |
-| ------- | ------------ | ------- |
-| 6       | Available    | ✓       |
+|---------|--------------|---------|
+| 8       | Available    |         |
+| 6       | Deprecated   | ✓       |
```

Comment:
maybe document v7 here as deprecated?

Reply:
v7 was technically never released

Reply:
It probably would be worth referencing it, simply because of how widely v7 is used (which was the whole reason v8 had to be made instead, to prevent breaking large numbers of existing applications backed by some of the most commonly used wrapper libraries, right?)